.changelog/v0.3.0.toml

```diff
@@ -3,8 +3,8 @@ title = ""
 description = ""
 
 [[features]]
-title = ""
-description = ""
+title = "Switch Port LLDP Neighbors"
+description = "CRUD functionality for LLDP neighbors seen on a switch port. [#263](https://github.com/oxidecomputer/oxide.go/pull/263)"
```

Comment:
Should we mention support bundles, too?

Reply:
These aren't available for the Go SDK since they have a "hidden" [tag](https://github.com/oxidecomputer/omicron/blob/main/openapi/nexus.json#L306-L308). This SDK doesnt include hidden endpoints. Thanks for bringing it up though!